release-pdf.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I found it useful to always add [`set -e`](https://www.gnu.org/software/bash/manual/bashref.html#The-Set-Builtin) so that one command called by the script fails, the script will exit.